src/Radar.js

```diff
@@ -53,16 +53,30 @@ export default class Radar extends Viz {
     const height = this._height - this._margin.top - this._margin.bottom,
           width = this._width - this._margin.left - this._margin.right;
 
-    const radius = (Math.min(height, width) - this._radarPadding) / 2,
+    const depth = this._depth || 0,
+          radius = (Math.min(height, width) - this._radarPadding) / 2,
           transform = `translate(${width / 2}, ${height / 2})`;
 
-    const maxValue = Math.max(...this._filteredData.map((d, i) => this._value(d, i))),
-          nestedAxisData = nest()
-        .key(this._y)
+    const nestedAxisData = nest()
+        .key(this._metric)
         .entries(this._filteredData),
           nestedGroupData = nest()
-        .key(this._x)
+        .key(this._groupBy[depth])
+        .key(this._metric)
         .entries(this._filteredData);
+        
+    const maxValue = Math.max(...nestedGroupData.map(h => h.values.map(d => d.values.reduce((sum, x, i) => sum + this._value(x, i), 0))).flat());
```

Comment:
Might `import {max, sum} from "d3-array";` be better suited here? You pass them the data array and an optional accessor.